compiler/rustc_trait_selection/src/error_reporting/traits/on_unimplemented_format.rs

```diff
@@ -376,39 +376,4 @@ pub mod errors {
     #[diag(trait_selection_missing_options_for_on_unimplemented_attr)]
     #[help]
     pub struct MissingOptionsForOnUnimplementedAttr;
-
-    #[derive(LintDiagnostic)]
-    #[diag(trait_selection_ignored_diagnostic_option)]
-    pub struct IgnoredDiagnosticOption {
-        pub option_name: &'static str,
-        #[label]
-        pub span: Span,
-        #[label(trait_selection_other_label)]
-        pub prev_span: Span,
-    }
```

Comment:
This is a duplicate of the one in `../on_unimplemented.rs`, I think I accidentally copy pasted it in the past 😞